B827EBFFFEBEB601.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFEBEB601",
+    "servers": [
+      {
+        "server_address": "ttn-router-asia-se",
```

Comment:
The router address is not valid, the current list of address is here: https://www.thethingsnetwork.org/wiki/Backend/Connect/Gateway